bin/octokit-schema.js

```diff
@@ -4,25 +4,37 @@ const path = require("path");
 const prettier = require("prettier");
 const $RefParser = require("@apidevtools/json-schema-ref-parser");
 
+const pathToWebhookSchemas = "payload-schemas/schemas";
+
 const removeExtension = (fileName, ext) => {
   assert.ok(fileName.endsWith(ext), `"${fileName}" does not end with "${ext}"`);
 
   return fileName.substring(0, fileName.length - ext.length);
 };
 
 const buildCommonSchemasDefinitionSchema = () => {
-  const commonSchemas = fs.readdirSync("payload-schemas/schemas/common");
+  const commonSchemas = fs.readdirSync(`${pathToWebhookSchemas}/common`);
   const definitions = {};
 
   commonSchemas.forEach((schema) => {
     definitions[removeExtension(schema, ".schema.json")] = {
-      $ref: `payload-schemas/schemas/common/${schema}`,
+      $ref: `${pathToWebhookSchemas}/common/${schema}`,
     };
   });
 
   return definitions;
 };
 
+const listEvents = () => {
+  const dirents = fs.readdirSync(pathToWebhookSchemas, {
```

Comment:
what is a dirent? Maybe call the variable `files`? Or something more descriptive? I prefer to avoid any kind of abbreviations to make the code easier to read / maintain / contribute to

Reply:
It's a [directory entity](https://nodejs.org/api/fs.html#fs_class_fs_dirent), which is what `readdirSync` returns when `withFileTypes` is `true`.

I agree with avoiding abbreviations, but in this case it's the name of the actual underlying class and the expanded version is very long `directoryEntity` - the reason I don't want to call it something like `files` is that it's not a file: it has methods on it that you can call.

Reply:
`directoryEntity` is not long, you can call it `directoryEntities` or `directoryEntries`. I was just confused by `dirents`, I assume others will, too

Reply:
sorry your right here it'd be called `directoryEntities` - but does that mean you're happy for `dirent` to be used for the `filter` & `map` parameters?

Reply:
I'd call it `directoryEntity` in both, or `entity` would also work in this context, better than `dirent`, I think